### DISCUSSION TITLE
Volume controls on Grid mode

### DISCUSSION THREAD

**Discussion by soymonitus** [SynthstromAudible/DelugeFirmware#749] [Ideas]
This is for @PaulFreund  Given that the new Grid mode is vertical, and all the tracks (with their corresponding variations) are set vertically, and we have the audition pads to select the Mode.... It would be awesome to have a third mode button that enables vertical faders (like the Automation View), one per track. With this you can control (and overwrite) the Master Volume of each of the tracks (affectall volume for kits tracks). That way you have like a mixer with track levels.

**Comment by PaulFreund** [comment]
Yes fully agree, I planned to introduce something like this as yellow mode but we need to figure out what should happen internally. The track (output in the code) does not have it's own volume or offset so we can either introduce one which will make gain staging less obvious or change the volume of all clips which might also confuse a bit. Suggestions welcome :)
